### PR TITLE
Remove a couple of unused helper functions

### PR DESCRIPTION
**Pull request by markusthoemmes**
As per title. These aren't in use anywhere.

### CODE REVIEW

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: hack/lib/common.bash [hunk 1]
```diff
@@ -1,26 +1,11 @@
 #!/usr/bin/env bash
 
-function array.contains {
```
Comment (cardil) [inline hack/lib/common.bash:3]: I'm using that in eventing upgrade tests. Please don't remove it.

Comment (cardil) [reply hack/lib/common.bash:3]: /hold

Comment (markusthoemmes) [reply hack/lib/common.bash]: Git would've just given it back to you anyway :shrug:. Pushed.

Comment (cardil) [reply hack/lib/common.bash]: Sure. Just need to rebase and wait for tests. Thanks.

**Pull request merged**